Bafc5 305 spaces

va5y4 opened the pull request.

va5y4 committed:
add container class

va5y4 committed:
update body layout

va5y4 committed:
style header container

va5y4 committed:
refactor layout structure and styles for consistency

va5y4 committed:
refactor layout structure and styles for hero section and base template

va5y4 committed:
Merge branch 'main' of https://github.com/bro-academy/binabox-va5y4 into BAFC5-305-spaces

va5y4 committed:
adjust box model

va5y4 committed:
refactor hero component structure and styles for improved layout

va5y4 committed:
refactor styles for blog card, catalog, employee, hero, and product components

Commit:
refactor blog section template and styles for improved layout and readability

va5y4 committed:
refactor sidebar styles

va5y4 committed:
refactor article and sidebar styles for improved layout and organization

va5y4 committed:
refactor article styles and update blog article template for improved layout and readability

va5y4 committed:
refactor article styles and update blog article template for improved layout and organization

va5y4 committed:
refactor footer layout and styles for improved organization and readability

va5y4 committed:
refactor footer structure and styles for improved layout and organization

va5y4 committed:
upd

va5y4 committed:
refactor footer and catalog sidebar styles for improved layout and consistency

va5y4 committed:
upd

Kapustin3107 commented:
1. Добавить вертикальный паддинг и паддинг справа как по макету на search input. Проверить аналогичный input на других частях сайта. Также предлагаю вынести стили для полей ввода в отдельный BEM блок и все  стили записать в отдельном scss файле.

<img width="1077" height="424" alt="Screenshot 2025-11-13 at 22 42 49" src="https://github.com/user-attachments/assets/db6d8b10-60c4-45dd-b2e0-cbde13ee50cd" />

2. Плохая практика использовать отрицательный margin между элементами. Прошу исправить )

<img width="1518" height="686" alt="Screenshot 2025-11-13 at 22 54 36" src="https://github.com/user-attachments/assets/3a2dae30-7906-4ca9-b2a7-49d3d0b950f8" />

3. Не правильно задан отступ между текстом и числами

<img width="851" height="345" alt="Screenshot 2025-11-13 at 22 59 00" src="https://github.com/user-attachments/assets/213042f0-aab9-4ba5-9852-f7a336538260" />

4. В карточке товаров выделенные на скриншоте элементы должны находится внутри ссылки. Так же проверь отступы внутри карточки

<img width="953" height="425" alt="Screenshot 2025-11-13 at 23 20 57" src="https://github.com/user-attachments/assets/eeaa1bb6-6e6a-4b92-8f3c-c880ed345e04" />

5. В карточках блога на главной странице отсутствуют paddings

<img width="775" height="830" alt="Screenshot 2025-11-13 at 23 24 39" src="https://github.com/user-attachments/assets/b8accea7-9494-42f5-a2e7-d212012db45d" />

6. Слишком большое расстояние между элементами

<img width="723" height="100" alt="Screenshot 2025-11-13 at 23 25 35" src="https://github.com/user-attachments/assets/068e6d7c-62f7-42b5-aafc-b66d69122704" />

7. Не правильный padding на кнопках

<img width="280" height="193" alt="Screenshot 2025-11-13 at 23 28 12" src="https://github.com/user-attachments/assets/fe3ec019-c8b0-4666-b4b4-9b400e5d482b" />

8. Пропали стили с карточек для блока About Us

<img width="1381" height="768" alt="Screenshot 2025-11-13 at 23 32 53" src="https://github.com/user-attachments/assets/f48a63c5-f047-435e-a95f-74f8bf0b432e" />

9. Добавить отступы на карточки в блоке Our team. Так же не правильные отступы между элементами внутри этой карточки

<img width="409" height="430" alt="Screenshot 2025-11-13 at 23 35 17" src="https://github.com/user-attachments/assets/193566d8-4313-4ffe-9351-c4ea9aa47d01" />

10. Не правильные отступы внутри FAQ элементов

<img width="1702" height="346" alt="Screenshot 2025-11-13 at 23 38 57" src="https://github.com/user-attachments/assets/fe2c9318-993b-491c-8b46-c306c2c89547" />

11. Отступы внутри footer тоже не по макету

Вывод. Нужно пройтись по всем секциям главной страницы и проверить все отступы (внутренние и внешние) на соответствие макету в Figma.

Kapustin3107 reviewed (changes requested):
Рекомендую доработать )

va5y4 committed:
match figma search

va5y4 committed:
add search input component and update header and search form templates

va5y4 committed:
upd

va5y4 committed:
add: flex

va5y4 committed:
remove negative margins

va5y4 committed:
style: update quote styles and structure in article component

va5y4 committed:
style: refactor hero component layout and remove negative margins

va5y4 committed:
style: reorder product card template for improved readability

va5y4 committed:
style: enhance product card layout and improve class naming for consistency

va5y4 committed:
style: restructure product card layout for improved organization and readability

va5y4 committed:
style: adjust blog card layout and improve content structure for better readability

va5y4 committed:
upd read-more

va5y4 committed:
Enhance blog card component with author name styling and date icon

- Added a new SCSS class for uppercase author names in blog cards.
- Updated blog card Twig templates to include a date icon next to the publication date.
- Ensured consistent usage of the icon function across blog card templates.

va5y4 committed:
style: enhance about section layout and improve feature presentation

va5y4 committed:
style: update employee component layout for improved structure and readability

va5y4 committed:
style: refine FAQ component layout for improved spacing and readability

va5y4 committed:
style: improve footer layout with new bottom container and adjust padding

va5y4 committed:
style: update catalog, header, product, separator, blog, and template styles for improved layout and consistency
